app/Domains/Contact/ManageContact/Web/ViewHelpers/ContactShowViewHelper.php

```diff
@@ -285,6 +286,10 @@ public static function modules(TemplatePage $page, Contact $contact, User $user)
                 $data = ModulePhotosViewHelper::data($contact);
             }
 
+            if ($module->type == Module::TYPE_POSTS) {
```

Comment:
I would have used a `switch` on the whole foreach, but it's more changes

Reply:
You are right. I won't do this in this PR but will do it. It'll be faster.